xmppserver/src/main/webapp/session-details.jsp

```diff
@@ -356,6 +356,21 @@
             Invalid session/connection
                 <% }
             } %>
+        </td>
+    </tr>
+    <tr>
+        <td class="c1">
+            <fmt:message key="session.details.software_version"/>
+        </td>
+        <td>
+                <%
+                try { %>
+                <%= currentSess.getSoftwareVersion().get("name") %> / <%= currentSess.getSoftwareVersion().get("version") %> / <%= currentSess.getSoftwareVersion().get("os") %>
```

Comment:
It is important to properly escape the values that are being displayed, to avoid malicious user to be able to inject things in the admin console pages!